### PR TITLE
Inline idl overmatch

### LINKED ISSUE #2258: Overmatching on enum values when two are on the same line
## Description of problem
Given: 
```HTML
<pre class="idl">
   enum WakeLockType { "screen", "system" };
</pre>
<p>{{ WakeLockType["screen"] }} and {{ WakeLockType["system"] }}</p>
```

It overmatches and gets a syntax error.

### PR DESCRIPTION
**Pull request by marcoscaceres**
fixes #2258 

Kinda ugly... can anyone think of a better solution?

### CODE REVIEW

**Comment by sidvishnoi** [comment]
``` js
const re = /{{[^}}]*}}/
```

image

**Comment by marcoscaceres** [comment]
reading https://mariusschulz.com/blog/why-using-in-regular-expressions-is-almost-never-what-you-actually-want#lazy-quantifiers-to-the-rescue

File: src/core/inlines.js [hunk 1]
```diff
@@ -147,7 +147,7 @@ export function run(conf) {
       "\\b(?:NOT\\s+)?REQUIRED\\b",
       "\\b(?:NOT\\s+)?RECOMMENDED\\b",
       "\\bOPTIONAL\\b",
-      "(?:{{2}\\s*.*\\s*}{2})", // inline IDL references,
+      "(?:{{[^}}]+}})", // inline IDL references,
```
Comment (saschanaz) [inline src/core/inlines.js]: As @sidvishnoi said this should really be `[^}]`, because exclude `}` twice is redundant.

Comment (marcoscaceres) [reply src/core/inlines.js]: oh whoops. Merged too quick.

Comment (sidvishnoi) [reply src/core/inlines.js]: I'll send fix with https://github.com/w3c/respec/issues/2259

**Pull request merged**